doc/project/dev_env_and_tools.rst

```diff
@@ -258,138 +258,165 @@ fix it.
-|``EXT``                 | Related to an external component (in ``ext/``)      |
-+------------------------+-----------------------------------------------------+
-
-PR only labels
---------------
-
-================ ===============================================================
-``DNM``          This PR should not be merged (Do Not Merge).
-                 For work in progress, GitHub "draft" PRs are preferred
-``Stale PR``     PR which seems abandoned, and requires attention by the author
-``Needs review`` The PR needs attention from the maintainers
-``Backport``     The PR is a backport or should be backported
-``Licensing``    The PR has licensing issues which require a licensing expert to
-                 review it
-================ ===============================================================
-
-Issue only labels
------------------
+* ``Regression``
+
+Something, which was working, but does not anymore (bug subtype).
+
+* ``Question``
+
+This issue is a question to the Zephyr developers.
+
+* ``Enhancement``
+
+Changes/Updates/Additions to existing :ref:`features <feature-tracking>`.
+
+* ``Feature request``
+
+A request for a new :ref:`feature <feature-tracking>`.
+
+* ``Feature``
+
+A :ref:`planned feature<feature-tracking>` with a milestone.
+
+* ``Duplicate``
+
+This issue is a duplicate of another issue (please specify).
+
+* ``Good first issue``
+
+Good for a first time contributor to take.
+
+* ``Release Notes``
 
-==================== ===========================================================
-``Regression``       Something, which was working, but does not anymore
-                     (bug subtype)
-``Question``         This issue is a question to the Zephyr developers
-``Enhancement``      Changes/Updates/Additions to existing
-                     :ref:`features <feature-tracking>`
-``Feature request``  A request for a new :ref:`feature <feature-tracking>`
-``Feature``          A :ref:`planned feature<feature-tracking>` with a milestone
-``Duplicate``        This issue is a duplicate of another issue
-                     (please specify)
-``Good first issue`` Good for a first time contributor to take
-``Release Notes``    Issues that need to be mentioned in release notes as known
-                     issues with additional information
-==================== ===========================================================
+Issues that need to be mentioned in release notes as known issues with
+additional information.
 
 Any issue must be classified and labeled as either ``Bug``, ``Question``,
 ``Enhancement``, ``Feature``, or ``Feature Request``. More information on how
 feature requests are handled and become features can be found in
 :ref:`Feature Tracking<feature-tracking>`.
+
+Labels applicable to pull requests only
+=======================================
+
+The issue or PR describes a change to a stable API. See additional information
+in :ref:`stable_api_changes`.
```

Comment:
This doesn't belong here. This is a description of the ``Stable API Change`` label. It should be removed from here.

Reply:
I think it is now fixed in https://github.com/zephyrproject-rtos/zephyr/pull/44774/commits/a35f8a0c5d4be4044ed1036eb1603db4c6c2c352 .